_posts/2020/2020-03-24-hr-post-2.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title: Get in touch with HR
```

Comment:
This title might be a bit too short and difficult to understand. What about "HR insights - ePages at job fairs" ?